librz/util/buf.c

```diff
@@ -430,6 +648,15 @@ RZ_API RzBuffer *rz_buf_ref(RzBuffer *b) {
 	return b;
 }
 
+/**
+ * \brief Append the content of the buffer a to the buffer b.
+ * \param b A buffer with write permission.
+ * \param a ...
```

Comment:
\param dst the destination buffer with write permission
\param src the source buffer